.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 *.backup
 __pycache__
+front/tests/unit/coverage
```

Comment:
missing new line

Reply:
You probably want to configure you're editor. For vscode:

```json
{
  "files.insertFinalNewline": true,
}
```

Reply:
yup, we can also use the [.editorconfig standard](https://editorconfig.org) to autoconfigure any IDE 

```
# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true
```

Reply:
updated with a new line in the end